packages/integrations/mdx/CHANGELOG.md

```diff
@@ -2,19 +2,17 @@
 
 ## 0.15.0
 
-### Minor Changes
+**Minor Changes**
```

Comment:
```suggestion
### Minor Changes
```
Not sure why this is a bold header while others remain h3s?

Reply:
Good catch — thank you! Intentionally removed the h4s _inside_ a specific change bullet, but these should still be h3s.